pages/Badge/badge.html

```diff
@@ -0,0 +1,164 @@
+                        sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                  <h2 class="text-heading">Badge on Text</h2>
+                  <div class="text-container">
+                        <span class="text">This is small text</span>
+                        <span class="small-badge">5</span>
+                        <span class="medium-text">This is medium text</span>
+                        <span class="medium-badge">5</span>
+                        <span class="medium-text">Button with a badge on the left<button class="action-button">
```

Comment:
Try giving the button a position relative and then the badge a position absolute so that you can position the badge properly on a corner